pyproject.toml

```diff
@@ -34,10 +34,13 @@ dynamic = [
     "readme",
     "version",
 ]
-description = "GRIB interface for Iris"
+description = "Functionality for converting between weather/climate datasets stored as GRIB files and SciTools-Iris Cubes"
 keywords = [
     "iris",
     "GRIB",
+    "data-analysis",
+    "earth-science",
+    "meteorology",
```

Comment:
A selection of keywords from the `pyproject.toml` in SciTools/iris ([link](https://github.com/SciTools/iris/blob/e26bfaf93dd06f82f195ee303960dc3ae03b1d09/pyproject.toml#L39-L50)).